.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Not super important but this doesn't look like a lockfile or autogenerated?

Reply:
this file was generated when running `yarn policies set-version`